packages/types/src/items.ts

```diff
@@ -62,6 +67,11 @@ export type MutationOptions = {
 	 */
 	bypassLimits?: boolean | undefined;
 
+	/**
+	 * Skips the creation of accountability and revision entries
+	 */
+	bypassAccountability?: boolean | undefined;
```

Comment:
It might be a good idea to rename this to something more descriptive (e.g., `skipTracking`, `skipAuditLog`, or `bypassAuditLog`). It currently not clear that it is a flag for bypasses tracking instead of user accountability.